playbooks/templates/standalone_parameters.yaml.j2

```
@@ -69,6 +69,12 @@ parameter_defaults:
   CephPoolDefaultPgNum: 8
   CephPoolDefaultSize: 1
 {% endif %}
+{% if standalone_extra_config|length > 0 %}
+  StandaloneExtraConfig: {{ standalone_extra_config }}
+  {% for key, value in standalone_extra_config.iteritems() %}
```

This is python2 syntax and fails in the `Create standalone_parameters.yaml` task:

```
TASK [Create standalone_parameters.yaml] *********************************************************************************************************************************
fatal: [standalone]: FAILED! => {"changed": false, "msg": "AnsibleUndefinedVariable: 'dict object' has no attribute 'iteritems'"}                                         
```


```suggestion
  {% for key, value in standalone_extra_config.items() %}
```